docs/glossary.md

```diff
@@ -21,17 +27,16 @@ Collection, roughly speaking, is a set of data. A set of structured data is ofte
 
 ### Commit
 A commit is a point-in-time snapshot of the branch. It's a collection of object metadata and data, including paths and the object contents and metadata. Commits have their own commit metadata. Every repository has one initial commit with no parent commits. If a commit has more than one parent, it is a merge commit. lakeFS supports only merge commits with two parents.
-
```

Comment:
Please don't remove blank lines before section headings.  This is actually [unclear Markdown](https://www.markdownguide.org/basic-syntax/#heading-best-practices).  While the GitHub Markdown processor does handle it as we intend, other processors might not -- and it increases ambiguity about what we _really_ intended to happen!